airbyte-integrations/connectors/destination-pinecone/integration_tests/pinecone_integration_test.py

```diff
@@ -0,0 +1,76 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import json
+import logging
+
+import pinecone
+from airbyte_cdk.models import DestinationSyncMode, Status
+from destination_langchain.destination import DestinationPinecone
+from destination_langchain.embedder import OPEN_AI_VECTOR_SIZE
+from integration_tests.base_integration_test import BaseIntegrationTest
+from langchain.embeddings import OpenAIEmbeddings
+from langchain.vectorstores import Pinecone
+
+
+class PineconeIntegrationTest(BaseIntegrationTest):
+    def _init_pinecone(self):
+        pinecone.init(api_key=self.config["indexing"]["pinecone_key"], environment=self.config["indexing"]["pinecone_environment"])
```

Comment:
For these tests I need to put some credentials into our CI secret store. Could you set up a stable environment somewhere we can test against (the test will run once a day or during development on the connector itself)

Reply:
I created a couple of dedicated indexes for tested:

OpenAI - https://airbyte-integration-openai-82836bc.svc.us-east-1-aws.pinecone.io
Cohere - https://airbyte-integration-cohere-82836bc.svc.us-east-1-aws.pinecone.io

I'll share credentials offline.